packages/cli/test/unit/commands/ps/index.unit.test.ts

```diff
@@ -0,0 +1,279 @@
+import {stdout, stderr} from 'stdout-stderr'
+import Cmd  from '../../../../src/commands/ps'
+import runCommand from '../../../helpers/runCommand'
+const nock = require('nock')
+const {expect} = require('chai')
```

Comment:
Example of old test setup still needing to be migrated to fancy test in the following format:

```
test
  .stderr()
  .stdout()
  .nock()
  .it('example test description', ({stdout, stderr}) => {
    expect(stdout).to.equal('foo')
    expect(stderr).to.equal('bar')
    ...
  })